examples/analytics/spark-k8s-operator/main.tf

```diff
@@ -221,66 +128,7 @@ module "eks_blueprints_kubernetes_addons" {
   eks_oidc_provider    = module.eks_blueprints.oidc_provider
   eks_cluster_version  = module.eks_blueprints.eks_cluster_version
 
-  #---------------------------------------------------------------
```

Comment:
These might have been added in `main` recently and we'll want to carry these over